chain-impl-mockchain/Cargo.toml

```diff
@@ -39,4 +39,5 @@ chain-crypto = { path = "../chain-crypto", features=["property-test-api"]}
 chain-addr = { path = "../chain-addr", features=["property-test-api"]}
 ed25519-bip32 = "0.1"
 rand_chacha = "0.1"
+rand = "0.6.0"
```

Comment:
I think @vincenthz split the rand deps partly to improve build times. Is it possible to depend on smaller sub-crates here?

Reply:
yes, but more to the case, this should use quickcheck arbitrary so that we make sure to have a deterministic result, not `rand`